src/components/ViewFieldIngelogdExtraInfo/ViewFieldIngelogdExtraInfo.js

```diff
@@ -38,6 +42,13 @@ const ViewFieldIngelogdExtraInfo = ({ crudObject, hideEdit, className }) => {
         setEigenaren(newEigenaren)
     }, [crudObject])
 
+    /**
+     * Function to get all users (admins) that can edit.
```

Comment:
`to get all users` is a bit confusing, as it only checks if the currently auth user has the rights
-> 
`Returns true if the user is authenticated to edit this policy object`